### PR TITLE
Update function create folder

### PR DESCRIPTION
**Pull request by dixyushi**

### CODE REVIEW

File: documentation/Functions.md [hunk 1]
```diff
@@ -244,11 +249,18 @@ The option to insert at a linenumber uses a placeholder inserted by a script and
 ***
 
 ### createFolder <a name="createFolder"></a>
+This function is used when you want to use an other command in a location which does not exist at the moment
```
Comment (GuentherJulian) [inline documentation/Functions.md:252]: I would simply write: This function creates a new folder
And then you can mention that one can use this when wanting to run other commands (e.g. createFile) in locations that does not exist at the moment

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md [hunk 2]
```diff
@@ -244,11 +249,18 @@ The option to insert at a linenumber uses a placeholder inserted by a script and
 ***
 
 ### createFolder <a name="createFolder"></a>
+This function is used when you want to use an other command in a location which does not exist at the moment 
 #### parameter 
-1. Path of the folder to be created, relative to the workspace directory. Subdirectories are also created.
+You need to add only one parameter.
```
Comment (EduardKrieger) [inline documentation/Functions.md:254]: pass only one parameter

File: documentation/Functions.md [hunk 3]
```diff
@@ -244,11 +249,24 @@ The option to insert at a linenumber uses a placeholder inserted by a script and
 ***
 
 ### createFolder <a name="createFolder"></a>
+This function is used to create a new folder.
 #### parameter 
-1. Path of the folder to be created, relative to the workspace directory. Subdirectories are also created.
+This function has only one parameter.
```
Comment (EduardKrieger) [inline documentation/Functions.md:254]: For some functions you are using  `This function consist of two parameters.` this should be consistent.

**Review by EduardKrieger — CHANGES_REQUESTED** [review]

File: documentation/Functions.md [hunk 4]
```diff
@@ -244,11 +249,24 @@ The option to insert at a linenumber uses a placeholder inserted by a script and
...
+    - **Required**
+    - **Type**- String
+    - **Description**- Path of the folder to be created (relative to working directory). Subdirectories are also created.
+
 #### example 
 createFolder("directoryPath/subDirectory")
 
+Note:
+
+1. The command for execution will be generated by katacoda runner, so user will have to execute this command manually.
```
Comment (GuentherJulian) [inline documentation/Functions.md:266]: Katacoda

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md (hunk 4 above)
Comment (dixyushi) [reply documentation/Functions.md:266]: Done!!

**Pull request merged**